### ISSUE TITLE
Requires outdated "request" module

### ISSUE THREAD

**Issue by berlincount** [mykle1/MMM-PilotWX#7]

**Comment by scubasteve17** [comment]
I just installed request and it started working for me.
`npm install request`

**Comment by berlincount** [comment]
That's a (dangerous) workaround, not a fix.

The `request` module has been fully deprecated, there will be no security fixes or anything else coming.

See https://github.com/request/request/issues/3142

**Comment by scubasteve17** [comment]
True, it is EOL, but this is an older module and 'request' worked at the time. Though this was the only MMM that worked for me for grabbing METAR data.
Any others you recommend?
It's been a long while since I've worked with javascript, but might try to replace 'request' with 'got', based on this post:
https://nodesource.com/blog/express-going-into-maintenance-mode

I've also repurposed my RPi for something else so no urgency to test it for me.
